ui/tests/acceptance/secrets/backend/generic/secret-test.js

```diff
@@ -13,11 +13,11 @@ import showPage from 'vault/tests/pages/secrets/backend/kv/show';
 import listPage from 'vault/tests/pages/secrets/backend/list';
 import consolePanel from 'vault/tests/pages/components/console/ui-panel';
 import authPage from 'vault/tests/pages/auth';
-import { writeSecret } from 'vault/tests/helpers/kv/kv-run-commands';
-import { PAGE } from 'vault/tests/helpers/kv/kv-selectors';
+import { writeSecret } from 'vault/tests/helpers/kv/kv-selectors';
 
 import { create } from 'ember-cli-page-object';
 import { deleteEngineCmd, runCmd } from 'vault/tests/helpers/commands';
+import { KV_LIST } from 'vault/tests/helpers/components/kv/page/list-selectors';
```

Comment:
Since `page` components map to route templates I don't really think of them as components belonging in the `components/` directory, I would have expected these selectors to be a separate block in the `kv/kv-selectors` file. 

I had imagined that the `helpers/components/` would contain components that are reused all over the app.